c2rust-analyze/src/main.rs

```diff
@@ -1,4 +1,5 @@
 #![feature(rustc_private)]
+#[macro_use] extern crate assert_matches;
```

Comment:
I don't believe either `#[macro_use]` or `extern crate` are needed anymore.  I think `extern crate` is only needed for `rustc_private` crates.

Reply:
i'm not sure what you're suggesting, to use `use assert_matches`? `std::assert_matches::assert_matches` doesn't have the same functionality of this crate and the [docs](https://doc.rust-lang.org/std/assert_matches/macro.assert_matches.html) have it being used in this way.

Reply:
`#[macro_use]` is for Rust 2015.  It's not needed at all in Rust 2018 and 2021.  That just means the docs are very old.

Reply:
`extern crate` is also not needed anymore in most cases (except for things like `rustc_private`) in Rust 2018 and later.